src/main/java/com/hazelcast/cloud/Client.java

```diff
@@ -26,21 +22,21 @@ public static void main(String[] args) {
         ClientConfig config = new ClientConfig();
         config.getNetworkConfig().setRedoOperation(true);
         config.getNetworkConfig().getCloudConfig()
-            .setDiscoveryToken("YOUR_CLUSTER_DISCOVERY_TOKEN")
-            .setEnabled(true);
+                .setDiscoveryToken("YOUR_CLUSTER_DISCOVERY_TOKEN")
+                .setEnabled(true);
         config.setProperty("hazelcast.client.cloud.url", "YOUR_DISCOVERY_URL");
         config.setClusterName("YOUR_CLUSTER_NAME");
         HazelcastInstance client = HazelcastClient.newHazelcastClient(config);
 
         System.out.println("Connection Successful!");
 
-        // the 'mapExample' is an example with an infinite loop inside, so if you'd like to try other examples,
-        // don't forget to comment out the following line
         mapExample(client);
 
         //sqlExample(client);
```

Comment:
why comment out?

Reply:
it was like this before, I guess due to the fact that the `mapExample` was running in an infinite loop. now we can uncomment it if it is OK. @JakeSCahill WDYT?

Reply:
but, if we do this, we will not be able to have a consistent output for all clients, as c++ does not have SQL

Reply:
comment out for C++ for now until it is available?

Reply:
For C++, we want the same JSON data to be inserted.

We don't want to run all the samples at once. We just want to show connection and have the same data in a map.

Reply:
@JakeSCahill  Commenting out the code this way causes unused code warnings. 

If all you want is `just want to show connection and have the same data in a map called cities`, then `mapExample` should be enough. You are trying to show more things in this example.

Reply:
I meant, we don't need to say `don't forget to comment out the following line` any more. We can still keep the other samples and keep the invocations commented out.